README.md

```diff
@@ -1,16 +1,341 @@
 
 ## Introduction
 
-## SWBAT 1
+So far in JavaScript, we've discussed the "Three Pillars of Web Programming",
+**Recognizing JS events**, **Manipulating the DOM**, and **Communicate with the
+server**, and we saw how these pillars are incorporated into web applications.
+
+The process we've learned has been straightforward:
+
+1.  An HTML page renders on the screen
+2.  JavaScript is executed when the page loads
+3.  In the JavaScript, event listeners are created pointing to 'free-standing'
+    functions, listed out in the JavaScript file.
+
+- Some of these functions handle DOM manipulation
+- Some functions handle server communication
+- Still other functions may serve as 'helper' functions to others
+
+This set up works great when just getting off the ground; with a handful of
+functions, we can get an interactive website up and running. What happens,
+though, when we want to go beyond a small website?
+
+Let's say we want to add feature, so we write a few more functions. Another
+function to handle a new event. That function needs to call a separate function
+for some DOM manipulation, then maybe a third function for updating our server.
+Meanwhile, it turns out, we can write a helper function that not only helps the
+code we just wrote, but cleans up some previous code we've written.
+
+Then we decide to add another, and more functions are needed. As we use our
+website, we think of more and more things to add or adjust. Pretty soon, the
+code has grown quite long. What's worse, all of the different things that happen
+on our website are starting to get jumbled together in a heap of functions. You
+may be able to get everything working for every feature we add, but coding this
+way comes at a cost:
+
+**The more complicated our code gets, the harder it is to understand and
+change it.**
+
+In larger applications, we might be dealing with hundreds of functions, DOM
+elements, and events, all dependent on each other, forming a web of
+relationships. It becomes increasingly difficult to follow the flow of actions
+in your code as more and more functions are introduced. Changes to one function
+may have unforeseen affects on functions _'downstream'_.
+
+This web of dependent, free-standing functions can be improved with
+organization. We do things like group functions together, arrange code in a
+readable way. Our options, however, are still limited - its still a big web.
+
+In this and the following lessons, we're going to take a look at an alternative
+way to structure our code, Object Orientation. By understanding and using OO, we
+can _design_ our code to be easier to read, understand and change.
+
+## Define Object Orientation
+
+With Object Orientation, instead of a web, we can think of our code as a
+collection of _cells_. These cells are separated from each other, can contain
+information, data like variables, as well as behaviors, functions directly
+related to that data.
+
```

Comment:
This is not necessarily something that has to be addressed right now, but I think all these explanations would benefit from graphic illustration, or maybe just some gifs to break up the text.

Reply:
I agree